packages/blade/src/components/Checkbox/CheckboxLabel.d.ts

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import type { InputProps } from './useCheckbox';
+
+export * from './CheckboxLabel.web';
+
+export type CheckboxLabelProps = {
+  children: React.ReactNode;
+  /**
+   * Pass only on react-native
+   */
+  inputProps: Partial<Extract<InputProps, { onPress: any }>>;
```

Comment:
In react-native there's no input so we pass the `inputProps` to the CheckboxLabel on react-native. Which will cause it to become an interactive element. 

https://github.com/razorpay/blade/pull/544/files#diff-3058d8ae77970163278f0b01b1224b0cd42521626c10b6899076183a7000e082R173